packages/terra-site/src/examples/table/NoStripedTable.jsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import Table from 'terra-table';
+
+const shortid = require('shortid');
```

Comment:
+1 to Tyler's comment.  That can allow us to get rid of the additional dependency.